### ISSUE TITLE
Upgrade to support multiple child bridges.

### ISSUE THREAD

**Issue by kristiandg** [mganjoo/homebridge-leviton-home-away#2]
**Is your feature request related to a problem? Please describe:**
Ability to manage home/away for multiple homes on same plugin.

**Describe the solution you'd like:**
I'd like the ability to support multiple homes from the plugin. The easiest way I see to do this is by allowing the ability to create multiple child bridges (similar to pics below), which would allow the ability to name multiple homes (granted, that would be multiple logins to the Leviton portal, but it would be a quick/easy way to achieve it).

**Describe alternatives you've considered:**
Either expansion of the plugin to allow the creation of multiple homes, or the support for multiple child bridges.

**Additional context:**
In the below pic, if it could support adding additional child bridges, then you could easily log into the Leviton portal and call up two different homes.  However, I'd recommend the ability to rename the Home/Away "switch" that gets created, or else you'd have multiple switches simply called "Home Occupied". Or maybe the naming format could be "<HomeName> Occupied" so it simply incorporates the name of the Leviton site that you're querying.
<img width="813" alt="Screenshot 2024-07-11 at 12 29 15 PM" src="https://github.com/mganjoo/homebridge-leviton-home-away/assets/33264300/59feba10-71b0-41e1-bf76-d18f2cac52ad">

Wonderful plugin - great for automating the lighting with the alarm's Away status. :)

**Comment by mganjoo** [comment]
Adding multiple child bridges is a native feature of Homebridge, not something I would provide as part of this plugin. I think documentation on how to do this is a bit confusing (the UI seems to only allow setting up one child bridge), but it seems like you can do multiple child bridges via JSON by just adding two platform blocks: https://github.com/homebridge/homebridge/issues/3390#issuecomment-1620804598 has an example (haven't tried this myself though).

**Comment by kristiandg** [comment]
This is true, but it's my understanding the plugin itself needs like a line of code to enable the function to be supported for that plugin.

**Comment by mganjoo** [comment]
Can you point me to docs or a link describing the change required? Even better, happy to take a look at a PR that you put up that enables this.
